core/unit_test/TestTeamMDRange.hpp

```diff
@@ -169,7 +169,13 @@ struct TestTeamThreadMDRangeParallelFor : public TestTeamMDParallelFor {
     FillFlattenedIndex fillFlattenedIndex(leagueSize, n0, n1);
 
     Kokkos::parallel_for(
-        Kokkos::TeamPolicy<ExecSpace>(leagueSize, Kokkos::AUTO),
+        Kokkos::TeamPolicy<ExecSpace>(
+            leagueSize, Kokkos::AUTO
+#ifndef KOKKOS_ENABLE_OPENMPTARGET
+            ,
+            Kokkos::TeamPolicy<ExecSpace>::vector_length_max()
+#endif
+                ),
```

Comment:
We need at least 32 threads in a team and setting `vector_length_max` would give us 1 thread with `vector_length` 32.

Reply:
Does that happen even when you manually specify a number for `team_size`, or is it only when `Kokkos::AUTO` is used along with `vector_length_max`?

Reply:
We are now using `vector_length==2` if `OpenMPTarget` was enabled which seems to work fine as well.